util/chplenv/__init__.py

```diff
@@ -19,6 +19,7 @@
     'chpl_threads',
     'chpl_timers',
     'chpl_wide_pointers',
+    'utils',
```

Comment:
Without that I get this:
...
  File "/users/gbt/css/dev/chapel/util/printchplenv", line 48, in print_mode
    get_module_lcd = ( utils.using_chapel_module() and
NameError: global name 'utils' is not defined